ui/raidboss/data/05-shb/eureka/delubrum_reginae.js

```diff
@@ -547,6 +565,7 @@ export default {
           en: 'Stand In Opposite Meteor',
           de: 'Steh im entgegengesetztem Meteor',
           fr: 'Placez-vous au météore de l\'élément opposé',
+          cn: '吃相反温度的陨石',
```

Comment:
```suggestion
          cn: '接相反温度的陨石',
```

You can use 接, but avoid to use 吃 in something you **should** do. Here and everywhere.